modules/cudf/src/series/bool.ts

```diff
@@ -28,4 +30,97 @@ export class Bool8Series extends NumericSeries<Bool8> {
   get data() {
     return new Uint8ClampedBuffer(this._col.data).subarray(this.offset, this.offset + this.length);
   }
+
+  _prepare_scan_series(skipna: boolean) {
```

Comment:
Might be worth to set this to `true` as a default value.

Reply:
I'd like to make sure an explicit argument is always passed in to help ensure that the public API value (which is optional) is always used and passed on as expected.